README.md

```diff
@@ -12,6 +12,50 @@ A subset of available Janus IDP plugins is available at our [community site](htt
 
 You can also see the [Plugin Marketplace](https://backstage.io/plugins) for other open-source plugins you can add to your Backstage instance.
 
+## Dynamic Plugins Installation
+
+It is possible to install plugins without code changes in a backstage that supports [Dynamic Plugins](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md) (e.g. [Janus IDP](https://janus-idp.io/) and [Red Hat Developer Hub](https://developers.redhat.com/rhdh)). Follow the steps below to install a dynamic plugin:
```

Comment:
Are there any prerequisite for enabling the dynamic plugins?
I mean, can a user just use this procedure in a random version of Janus-IDP/Backstage, or should they first make sure a specific version is available, or anything perhaps something else?

Reply:
Hello @jkilzi I confirmed that no additional configuration is required to enable dynamic plugins. Regarding what version to use, I know that dynamic plugins are available in RHDH starting on 1.0, but for Janus IDP I recommend users to check if the endpoint `/api/dynamic-plugins-info/loaded-plugins`, if it is available, then you can use dynamic plugins.

Please bear in mind that upstream backstage does not support dynamic plugins ATM.